DOCS/man/ao.rst

```diff
@@ -138,11 +138,21 @@ Available audio output drivers are:
         passthrough (even if the device reports it as supported). Use with
         extreme care.
 
-
 ``coreaudio_exclusive`` (macOS only)
     Native macOS audio output driver using direct device access and
     exclusive mode (bypasses the sound server).
 
+``avfoundation`` (macOS only)
```

Comment:
since we have some TARGET_OS_IPHONE preprocessors included, does this also support iOS?

Reply:
Theoretically it supports iOS since the API is the same. However, I have no idea how to build mpv to iOS and cannot test it. I'd appreciate it if someone could help me test that.